tool_sideview/utils.py

```diff
@@ -1,13 +1,33 @@
-from qgis.core import QgsCoordinateReferenceSystem
-from qgis.core import QgsCoordinateTransform
-from qgis.core import QgsFeature
-from qgis.core import QgsGeometry
-from qgis.core import QgsPoint
-from qgis.core import QgsProject
-from qgis.core import QgsVectorLayer
-from qgis.core import QgsWkbTypes
-
 import math
+from qgis.core import (
```

Comment:
`make beautiful` zal dit weer terugdraaien.
Is het een instelling van pycharm die imports op deze manier doet?
Indien "ja", geef even een gil, want dan heb ik een reden om naar https://editorconfig.org te kijken.